keys.py

```diff
@@ -1,13 +1,18 @@
+import glob
 import os
 import shutil
 
 
 def copy(moddir):
-    keysdir = os.path.join(moddir, "keys")
-    if os.path.exists(keysdir):
-        for o in os.listdir(keysdir):
-            keyfile = os.path.join(keysdir, o)
-            if not os.path.isdir(keyfile):
-                shutil.copy2(keyfile, "/arma3/keys")
+    keys = glob.glob(os.path.join(moddir, "**/*.bikey"))
+    if len(keys) > 0:
+        for key in keys:
```

Comment:
could just be `for key in keys`, since if the length is zero nothing will happen anyway

Reply:
I suppose that's fair enough